tools/bazar/presentation/javascripts/form-edit-template/fields/image.js

```diff
@@ -28,13 +35,14 @@ export default {
   attributesMapping: {
     ...defaultMapping,
     ...{
-      1: 'name',
-      3: 'thumb_height',
-      4: 'thumb_width',
-      5: 'resize_height',
-      6: 'resize_width',
-      7: 'align'
-    }
+      1: "name",
+      3: "thumb_height",
+      4: "thumb_width",
+      5: "resize_height",
+      6: "resize_width",
+      7: "align",
+      8: "default_image",
```

Comment:
```suggestion
      13: "default_image",
```
Il faudrait éviter d'utiliser le numéro 8 car il est utiliser pour savoir si le champ est requis.
Le numéro 13 semble disponible pour le champ Image